charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -50,6 +50,7 @@ containerLogs:
       requests:
         cpu: 50m
         memory: 25Mi
+    priorityClassName: system-node-critical
```

Comment:
Can we set this to be empty by default? This will preserve existing behavior and will be opt in. Similar to [upstream otel operator](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-operator/values.yaml#L313)